website/docs/guides/container_service_extension_4_x_cluster_management.html.markdown

```diff
@@ -1,21 +1,21 @@
 ---
 layout: "vcd"
-page_title: "VMware Cloud Director: Container Service Extension v4.0 Kubernetes clusters management"
-sidebar_current: "docs-vcd-guides-cse-4-0-cluster-management"
+page_title: "VMware Cloud Director: Container Service Extension v4.1 Kubernetes clusters management"
+sidebar_current: "docs-vcd-guides-cse-4-x-cluster-management"
 description: |-
-  Provides guidance on provisioning Kubernetes clusters using Container Service Extension v4.0
+  Provides guidance on provisioning Kubernetes clusters using Container Service Extension v4.1
 ---
 
-# Container Service Extension v4.0 Kubernetes clusters management
+# Container Service Extension v4.1 Kubernetes clusters management
 
 ## About
 
-This guide explains how to create, update and delete TKGm clusters in a VCD appliance with Container Service Extension v4.0
+This guide explains how to create, update and delete TKGm clusters in a VCD appliance with Container Service Extension v4.1
```

Comment:
NIT, but maybe we should expand `TKGm` naming in one place for SEO and eye-scanning purposes. We know it by heart, but maybe others don't

Reply:
Changed, thanks!